Community-Calls.md

```diff
@@ -30,28 +30,42 @@ Some examples:
 **How are calls announced?**
 
 - An issue is created 1-2 months earlier
-- They appear on the [calendar](README.md#the-bitcoin-design-calendar)
+- They appear on the [calendar](https://bitcoin.design/contribute/#the-bitcoin-design-calendar)
 - Two @channel message are sent out on Slack, the first one 1 week ahead, and the second one 1 day ahead of the call
-- A regular Slack in the #general channel is sent out shortly before the call
+- A regular Slack post in the #general channel is sent out shortly before the call
+- For Twitter Spaces, the Space is shared via a Tweet when it starts
```

Comment:
You can schedule the Twitter space on twitter so people can subscribe and get notified via Twitter when its about to start also.